docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,312 @@ sidebar_position: 18
+   | `computeSubnetName` | The name of the subnet that will run the Private Operator. The value must match the value specified in [Set Up the VPC Network](#set-up-the-vpc-network). If you accepted the default, the value is `unified-id-subnet-operators`. |
+   | `count` | The count for the number of instances you want to bring up. The default is `2`. |
+
+2. Run the following command to deploy the ACIs:
+
+   ```
+   az deployment group create --name operator --resource-group {RESOURCE_GROUP_NAME} --parameters operator.parameters.json  --template-file operator.json
+   ```
+
+3. Get the IP addresses of the ACI instances you created by running the following command:
+
+   ```
+   az deployment group show -g {RESOURCE_GROUP_NAME} -n operator --query properties.outputs
+   ```
+
+   The output should look something like the following:
+   
+   ```
+   { "ipAddress": { "type": "Array", "value": [ "10.0.0.6", "10.0.0.5", "10.0.0.4" ] } }
+   ```
+
+### Set Up the Gateway Load Balancer
+
+The next step is to set up the [Gateway Load Balancer](https://learn.microsoft.com/en-us/azure/load-balancer/gateway-overview), which takes the private IP addresses of the ACIs you created and uses them as a [backend pool](https://learn.microsoft.com/en-us/azure/load-balancer/backend-pool-management).
+
+Follow these steps:
+
+1. Update the `gateway.parameters.json` file with the following required value:
+
+   | Parameter | Description |
+   | :--- | :--- |
+   | `containerGroupIPs` | The IP addresses of the ACI instances you created&#8212;the values output as a result of [Complete the UID2 Private Operator Setup](#complete-the-uid2-private-operator-setup) Step 4. |
+
+2. (Optional) If you don't want to accept the defaults, update the `gateway.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
+
+    Parameter | Description |
+   | :--- | :--- |
+   | `vnetName` | The virtual network name. The value must match the value specified in [Set Up the VPC Network](#set-up-the-vpc-network). If you accepted the default, the value is `unified-id-network`. |
+   | `gatewaySubnetName` | The name of the subnet that runs the UID2 Gateway. The value must match the value specified in [Set Up the VPC Network](#set-up-the-vpc-network). If you accepted the default, the value is `unified-id-subnet-gateway`. |
+   
+2. Run the following command:
+
+   ```
+   az deployment group create --name gateway --resource-group {RESOURCE_GROUP_NAME} --parameters gateway.parameters.json  --template-file gateway.json
+   ```
+
+3. Get the public IP address of the Gateway Load Balancer by running the following command:
+   
+   ```
+   az deployment group show -g {RESOURCE_GROUP_NAME} -n gateway--query properties.outputs
```

Comment:
There is a space missing which breaks the command. It should be:
```
az deployment group show -g {RESOURCE_GROUP_NAME} -n gateway --query properties.outputs
```

Reply:
Thank you! Fixed.